AppController/djinn.rb

```diff
@@ -3098,9 +3098,12 @@ def find_me_in_locations
         all_local_ips.each { |ip|
           if ip == node.private_ip
             @my_index = index
-            HelperFunctions.set_local_ip(node.private_ip)
             @my_public_ip = node.public_ip
             @my_private_ip = node.private_ip
+            if ip != HelperFunctions.local_ip
```

Comment:
This is kind of weird since `all_local_ips` is populated by `get_all_local_ips`. This `local_ip` method also uses `get_all_local_ips`. I think this would be a bit more understandable if we avoid the call to `local_ip` and just check the index.
```
all_local_ips.each_with_index { |ip, index|
  ...
  if index != 0
```

Reply:
well I could remove that check altogether: I wanted to leave a warning in case this code path is taken (I haven't seen it in a very long time to be honest) just in case we may pick the wrong IP in a multi-ethernet device configuration. Thoughts? should I just remove the check?

Reply:
I would be fine with either removing the warning log or removing the call to `local_ip`.

Reply:
I removed the use of local_ip (in this function, still one to go) and added a warning when mulitple IPs are detected.